background.ts

```diff
@@ -866,11 +866,14 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
       return ['no changes necessary', ''];
     }
 
-    const allowListConf = BackendHelper.createImageAllowListConf(cfg.containerEngine.imageAllowList);
-    const rcService = k8smanager.backend === 'wsl' ? 'wsl-service' : 'rc-service';
+    // Update image allow list patterns, just in case the backend doesn't need restarting
+    if (cfg.containerEngine.imageAllowList.enabled) {
```

Comment:
This section doesn't need to be conditional: we're only restarting `openresty` if it's started. (In general, I found that for our uses, always writing out unused config files is easier for debugging.) Though I can see how this is easier to handle the `nerdctl` script…